### DISCUSSION TITLE
VRAM on prompt processing

### DISCUSSION THREAD

**Discussion by fgdfgfthgr-fox** [ggml-org/llama.cpp#1618] [Q&A] [answered]
Hello there, does the size of VRAM have any effect on prompt processing? I see that in CuBlas the larger the batch size, the faster the prompt processing seems to get, however also consuming more VRAM. In that case, is it worth to get an RTX 2080ti 22G (which is available at ￥2800/$400USD in China) rather than a RTX 2080ti 11G (~￥1800/$250USD) just for the shake of loading larger models and faster prompt processing time?
Thank you.

**Comment by BetaDoggo** [comment, marked as answer]
Having more vram shouldn't effect prompt processing speed too much but like you said it will allow you to load larger models. At that price it seems like it would be a good idea to get the 22G card. My only concern would be the issues with support from the 3rd party modder in case the card dies prematurely or has other issues.

**Comment by mlv-** [comment]
> In that case, is it worth to get an RTX 2080ti 22G (which is available at ￥2800/$400USD in China) 

Would you mind linking where this model is being sold?

**Reply by fgdfgfthgr-fox** [reply]
Apparently, you can buy them in 闲鱼(not sure the English name of this) app, however it only ships to China.

**Reply by chenqiny** [reply]
can we sell them to the world?
